libraries/AP_PiccoloCAN/AP_PiccoloCAN.cpp

```diff
@@ -157,7 +157,7 @@ void AP_PiccoloCAN::loop()
             continue;
         }
 
-        timeout = AP_HAL::micros64() + 250;
```

Comment:
if changing to 32 bit micros, then we should use a 32 bit variable for timeout unless I've missed something subtle
actually, it looks like this will fail when board has been booted for more than 2^32 micros, which is 71 minutes. I really think this needs to be micros64(), and the 250 should be "250ULL"

Reply:
also, timeout is now only used in the loop, so should be local var